### DISCUSSION TITLE
full text search in book summary

### DISCUSSION THREAD

**Discussion by zpaolo11x** [gotson/komga#1684] [Ideas]
I don't think it's currently possible, but it would be useful if search could also look into the "summary" of books, maybe using a custom token to activate summary search

**Comment by gotson** [comment]
this would most likely yield a lot of false positive

**Reply by zpaolo11x** [reply]
What do you mean by "false positive"? Of course it depends on what you have in the summary. I have summaries of collection books with the titles of the stories in the book, and currently I have no way of looking for a specific title. Putting the search behind a token (e.g. summary:"farewell to weapons" ) would prevent false positives in "general" search.

**Reply by gotson** [reply]
for your specific use case it would work yes, but for everyone else searching by a few words in the summary would yield a lot of results totally unrelated, because of the way the search works.

can you maybe provide some examples on what you have in summary and what you would search for, as it is not very clear to me what the use case is for now

**Reply by zpaolo11x** [reply]
For example, there are many books by Junji Ito that are collections of short stories, each will have its own summary with the list of stories, like Smashed by Junji Ito it's summary would have something like:

Bloodsucking Darkness
Ghosts of Prime Time
Roar
Earthbound
Death Row Doorbell 
The Mystery of the Haunted House
The Mystery of the Haunted House: Soichi's Version
Soichi's Beloved Pet
 In Mirror Valley
I Don't Want to Be a Ghost 	
Library Vision
Splendid Shadow Song
Smashed

so if I wanted to look in a collection for the book that contains "In Mirror Valley" I would search with something like

summary:"mirror valley" 

and hopefully find that the volume is Smashed
